src/udev/scsi_id/scsi_serial.c

```diff
@@ -361,14 +468,37 @@ static int scsi_inquiry(struct scsi_id_device *dev_scsi, int fd,
 }
 
 /* Get list of supported EVPD pages */
-static int do_scsi_page0_inquiry(struct scsi_id_device *dev_scsi, int fd,
-                                 unsigned char *buffer, unsigned len) {
+static int sgio_do_scsi_page0_inquiry(struct scsi_id_device *dev_scsi, int fd,
+                                      unsigned char *buffer, unsigned len) {
+        int retval;
```

Comment:
Please use `int r;`. This is the common variable name for storing result of function calls in our code base.

Reply:
Done